### ISSUE TITLE
Wallaby executes the same test too many times in Jest project

### ISSUE THREAD

**Issue by endragor** [wallabyjs/public#118]
When I run Wallaby on my project, I get `Finished executing 32386 affected test(s)` when I just have around a hundred of tests at the moment. When I make a change to a test file that contains 2 tests, the log reads `Finished executing 864 affected test(s)`.

After enabling debug mode I saw that there are exactly 432 occurrences of `Test executed` log for each test. It also displays `Test name duplicate` 431 times for each of the two tests. Here is a snippet of debug log that I get after changing the test file: https://gist.github.com/endragor/50df7698955a8163cbfe

**Comment by ArtemGovorov** [comment]
Do you have something like dynamically created tests? For examples something like this:
``` JavaScript
describe('something', function() {
  for (var i = 0; i < 100; i++) {
    it('test', function () {
      //...
    });
  }
});
```

**Comment by endragor** [comment]
No.

**Comment by endragor** [comment]
Can now be reproduced in https://github.com/endragor/wallaby-jest-sample
The cause was setting `setupTestFrameworkScriptFile` in `wallaby.testFramework.configure`. I suppose it somehow conflicts with Wallaby's setup file.

**Comment by ArtemGovorov** [comment]
Awesome, thanks for the way to reproduce it. Fixed in core 1.0.49.

**Comment by endragor** [comment]
Great, thanks! Another one for your pleasure: #121
